packages/palette/package.json

```diff
@@ -116,6 +116,7 @@
     "@artsy/palette-tokens": "^2.6.0",
     "@seznam/compose-react-refs": "^1.0.6",
     "@styled-system/theme-get": "^5.1.2",
+    "lodash": "4.17.5",
```

Comment:
This is being pulled in in a few places as a transient dep, so properly added it

Reply:
We should not be using this in Palette